tests/data/multivariate_normal_data.py

```diff
@@ -0,0 +1,427 @@
+    space_args_list = [(n,) for n in n_list]
+    shape_list = [(n + n**2,) for n in n_list]
+    n_samples_list = [1, 3, 5]
+    n_points_list = [1, 3, 5]
+    n_vecs_list = [1, 3, 5]
+
+    def belongs_test_data(self):
+        n = self.n_list[0]
+        n_samples = self.n_samples_list[0]
+        random_data = [
+            dict(
+                n=n,
+                point=self.Space(n).random_point(n_samples=n_samples),
+                expected=gs.array([True] * n_samples),
+            )
+        ]
+        n = self.n_list[1]
+        n_samples = self.n_samples_list[1]
+        random_data.append(
+            dict(
+                n=n,
+                point=self.Space(n).random_point(n_samples=n_samples),
+                expected=gs.array([True] * n_samples),
+            )
+        )
+        n = self.n_list[2]
+        n_samples = self.n_samples_list[2]
+        point = gs.random.rand(n_samples, n + n**2)
+        random_data.append(
+            dict(n=n, point=point, expected=gs.array([False] * n_samples))
+        )
+        return self.generate_tests(random_data)
+
+    def random_point_shape_test_data(self):
+        random_data = list()
+        for n, n_samples in zip(self.n_list, self.n_samples_list):
+            expected = (n + n**2,) if n_samples == 1 else (n_samples, n + n**2)
+            random_data.append(
+                dict(point=self.Space(n).random_point(n_samples), expected=expected)
+            )
+        return self.generate_tests(random_data)
+
+    def sample_test_data(self):
+        random_data = list()
+        for n, n_samples in zip(self.n_list, self.n_samples_list):
+            for n_points in self.n_points_list:
+                if n_samples == 1:
+                    if n_points == 1:
+                        expected = (n,)
+                    else:
+                        expected = (n_points, n)
+                else:
+                    if n_points == 1:
+                        expected = (n_samples, n)
+                    else:
+                        expected = (n_points, n_samples, n)
+                random_data.append(
+                    dict(
+                        n=n,
+                        point=self.Space(n).random_point(n_points),
+                        n_samples=n_samples,
+                        expected=expected,
+                    )
+                )
+        return self.generate_tests(random_data)
+
+    def point_to_pdf_test_data(self):
+        random_data = list()
+        for n, n_samples in zip(self.n_list, self.n_samples_list):
+            for n_points in self.n_points_list:
+                random_data.append(
+                    dict(
+                        n=n,
+                        point=self.Space(n).random_point(n_points),
+                        n_samples=n_samples,
+                    )
+                )
+        return self.generate_tests([], random_data)
+
+
+class MultivariateCenteredNormalMetricTestData(_RiemannianMetricTestData):
```

Comment:
Not necessarily for this PR (would be too much probably), but we could create a parent test _FisherRaoMetricTestData to factor out the tests that are the same across information metrics

Reply:
Same for _InformationManifoldTestData

Reply:
Yes I agree, something to do.